sql/99-patches.sql

```diff
@@ -294,5 +294,74 @@ $P$
 );
 
 
+--------------------------------------------------------------------------------
+-- protect_reference column VARCHAR(100) > VARCHAR(255)
+--
+-- For tables bde.crs_title & bde.crs_ttl_inst_protect
+-- as per https://github.com/linz/linz-bde-uploader/issues/279
+--------------------------------------------------------------------------------
+
+PERFORM _patches.apply_patch(
+    'protect_reference column VARCHAR(100) > VARCHAR(255)',
```

Comment:
We should add a version prefix to this patch comment. I see @strk is using LOL version for the last few. Before that we used the up-coming BDE processor release number.

Reply:
Last BDE deployment version was 1.9

Reply:
So I guess next one should be 1.9.1 for a bug fix?

Reply:
next version of BDE processor is 1.10

Reply:
Or should we use the bde-schema version (1.12.1 or more likely 1.13)

Reply:
Jeremy refers to LOL version, not BDE version. I guess we could use the version of LOL which first happened to contain a value longer than 100 for that title reference

Reply:
So `LOL 3.22b: ` then? Seems okay to me.

Note: the last patch looks like it had a typo `LOL 3.22p: `